server/controllers/TripController.js

```diff
@@ -124,9 +124,8 @@ class TripController {
       delete updatedTripDTO.userId
       return updatedTripDTO
     } catch (error) {
-      if (config.server.env === 'DEV')
-        console.error('Error while updating trip:', error)
-      throw new Error('Could not edit trip')
+      error.message = 'Could not create trip: ' + error.message
+      throw error
     }
```

Comment:
reason i put if here is to only provide limited error message to the frontend, any reason to remove it?

Reply:
Resolved in discord chat!